packages/hs-react-ui/src/components/Label/Label.tsx

```diff
@@ -1,25 +1,83 @@
 import React from 'react';
 import styled from 'styled-components';
-
+import Icon from '@mdi/react';
+import { mdiCheckBold, mdiAsterisk } from '@mdi/js';
 import colors from '../../enums/colors';
 
 export const StyledLabel = styled.label`
-  ${({ color = colors.grayDark }: { color: colors | string }) => `
-    display: block;
+  ${({ color = colors.grayLight }: { color: colors | string }) => `
+    display: inline-flex;
     color: ${color};
-    text-transform: uppercase;
+    font-family: Arial, Helvetica, sans-serif;
```

Comment:
If you look at some of the other files, you'll see the `{fonts.body}` style - we should use that going forward to get the correct fonts.

Reply:
Had to bring in master to get the fonts.body enum